core/src/main/scala/com/salesforce/op/evaluators/OpEvaluatorBase.scala

```diff
@@ -61,28 +61,28 @@ trait OpHasLabelCol[T <: FeatureType] extends Params {
 /**
  * Trait for predictionCol which contains all output results param
  */
-trait OpHasFullPredictionCol extends Params {
-  final val fullPredictionCol: Param[String] = new Param[String](this, "fullPredictionCol", "prediction column name")
+trait OpHasPredictionCol extends Params {
+  final val predictionCol: Param[String] = new Param[String](this, "fullPredictionCol", "prediction column name")
 
-  def setFullPredictionCol(value: String): this.type = set(fullPredictionCol, value)
-  def setFullPredictionCol(value: FeatureLike[Prediction]): this.type = setFullPredictionCol(value.name)
-  final def getFullPredictionCol: String = $(fullPredictionCol)
+  def setPredictionCol(value: String): this.type = set(predictionCol, value)
+  def setPredictionCol(value: FeatureLike[Prediction]): this.type = setPredictionCol(value.name)
+  final def getPredictionCol: String = $(predictionCol)
 }
 
 /**
- * Trait for predictionCol param
+ * Trait for internal flattened predictionCol param
  */
-trait OpHasPredictionCol[T <: FeatureType] extends Params {
-  final val predictionCol: Param[String] = new Param[String](this, "predictionCol", "prediction column name")
-  setDefault(predictionCol, "prediction")
+trait OpHasPredictionValueCol[T <: FeatureType] extends Params {
+  final val predictionValueCol: Param[String] = new Param[String](this, "predictionCol", "prediction column name")
```

Comment:
`"predictionCol"` -> `"predictionValueCol"`?

Reply:
yes and `fullPredictionCol` -> `prediction col`